config.example.php

```diff
@@ -18,7 +18,7 @@
 // db server port
 define('SYS_DB_PORT', 3306);
 // db scanner type
-define('SYS_DB_TYPE', 'rocketmap'); // 'rocketmap', 'realdevicemap', 'monocle-alt-mysql' or 'monocle-alt-pgsql'
+define('SYS_DB_TYPE', 'rocketmap'); // 'rocketmap', 'realdevicemap', 'monocle-alt-mysql' or 'monocle-alt-pgsql', 'mad'
```

Comment:
small typo, would be better to put the `or` before `mad` instead of before `monocle`